lua/neogit/lib/collection.lua

```diff
@@ -58,8 +58,15 @@ function M.reduce(tbl, func, ...)
 end
 
 function M.find(tbl, func)
-  for _, item in ipairs(tbl) do
-    if func(item) then return item end
+  for i, item in ipairs(tbl) do
+    if func(item, i) then return item, i end
+  end
+  return nil
+end
+
+function M.at(tbl, idx)
+  for i, item in ipairs(tbl) do
+    if i == idx then return item end
   end
   return nil
 end
```

Comment:
Is this any different from just doing `tbl[idx]`? :sweat_smile:

Reply:
true